deploy/stash.sh

```diff
@@ -311,6 +351,10 @@ if [ "$STASH_UNINSTALL" -eq 1 ]; then
   kubectl delete secret stash-apiserver-cert --namespace $PROMETHEUS_NAMESPACE || true
   # delete psp resources
   kubectl delete psp stash-operator-psp stash-backup-job stash-backupsession-cron stash-restore-job || true
+  # delete default functions
+  kubectl delete function update-status pvc-backup pvc-restore || true
```

Comment:
`kubectl delete function.stash.appscode.com`
`kubectl delete task.stash.appscode.com`

I think for common kind names like this, we need to use the full resource.apigroup